File: ios/ReservationApp/ReservationApp/Domain/Models/Meal.swift
//
//  Meal.swift
//  ReservationApp
//
//  Created by Mehmet Ali Kısacık on 13.11.2025.
//

import Foundation

struct Meal: Codable, Identifiable {
    let id: String // UUID
    let name: String
    let description: String?
    let kcal: Int?
    let price: Double?
    let categoryId: String
    let categoryName: String
    let restaurantId: String
    let restaurantName: String
    let imageUrl: String?
}

File: android/domain/src/main/java/com/reservationapp/domain/model/Meal.kt
package com.reservationapp.domain.model

data class Meal(
    val id: String,
    val name: String,
    val categoryId: String,
    val categoryName: String?,
    val restaurantId: String,
    val restaurantName: String?,
    val description: String?,
    val price: Double?,
    val kcal: Int?,
    val imageUrl: String?
)

